lib/fuel-auth.js

```diff
@@ -38,6 +38,7 @@ function FuelAuth(options) {
 	this.refreshToken = options.refreshToken;
 	this.scope        = options.scope;
 	this.version      = version;
+	this.accessType   = options.accessType || "offline";
```

Comment:
If this defaults to `'offline'`, then it's no longer possible to set it to an empty value (the default API behavior), as not providing it will result in `offline`, providing `null` will till result in `offline`, etc. Just don't provide a default value for users to opt-in to the offline mode.